internal/faustjs.org/blog/2023-03-01-faust-update-mar-01-2023.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Faust Update - March 1, 2023
+description: Updates on Faust development for Mar 1, 2023
+slug: faust-update-mar-1-2023
+authors:
+  - name: Chris Wiegman
+    title: Engineering Manager
+    url: https://github.com/ChrisWiegman
+    image_url: https://avatars.githubusercontent.com/u/394675?v=4
+hide_table_of_contents: true
+---
+
+It's been a little while since I've had a chance to write an update post and we've accomplished a lot in that time. We have new releases, new features and more documentation all ready to go and we're starting to push even that further than we ever have. Here's how it all breaks down:
```

Comment:
The "push even that further" wording is a little strange. Otherwise looks good."!